storage/src/main/scala/ch/epfl/bluebrain/nexus/storage/files/CopyFiles.scala

```diff
@@ -0,0 +1,49 @@
+package ch.epfl.bluebrain.nexus.storage.files
+
+import cats.data.NonEmptyList
+import cats.effect.{IO, Ref}
+import cats.implicits.catsSyntaxParallelTraverse1
+import ch.epfl.bluebrain.nexus.storage.StorageError.CopyOperationFailed
+import fs2.io.file.{CopyFlag, CopyFlags, Files, Path}
+
+trait CopyFiles {
+  def copyValidated(name: String, files: NonEmptyList[ValidatedCopyFile]): IO[Unit]
```

Comment:
It's probably cleared is this is bucket rather than name, I think there are other places where this is true also